modules/apps/site-initializer/site-initializer-raylife-ap/src/main/resources/site-initializer/fragments/agent-portal/fragments/layout/index.html

```diff
@@ -45,14 +44,16 @@
 
 	<div class="d-flex flex-column w-100">
 		<div class="bg-neutral-0 d-flex flex-column sticky-top w-100">
-			<div class="align-items-center d-flex page-header-container px-4 py-3">
-				<p class="font-weight-light h3 m-0 text-neutral-10" data-lfr-editable-id="header-title" data-lfr-editable-type="text">
-					<span class="font-weight-bolder h3">Hi, Hugo.</span> You're on track this month!
+			<div class="align-items-center d-flex justify-content-between page-header-container px-4 py-3">
+			<p class="font-weight-light h3 m-0 text-neutral-10" data-lfr-editable-id="header-title" data-lfr-editable-type="text">
+					<span class="font-weight-bolder h3">Hi, Hugo.</span> You're
+					on track this month!
 				</p>
+			<img alt="search" class="search-name" src="[resources:search_icon.svg]" />
 			</div>
 		</div>
 
-		<div class="raylife-agent-portal-content">
+		<div>
```

Comment:
@silasbsi you could maintain the class name here, because this will be helpful to style the content from React side, if needed

```suggestion
		<div class="raylife-agent-portal-content">
```